nuxt.config.js

```diff
@@ -141,6 +141,14 @@ export default {
         },
       },
     ],
+    [
+      "@nuxtjs/recaptcha",
+      {
+        hideBadge: true,
+        siteKey: "6LcduWcbAAAAAKCNVxolki6wVpCCpKNDVH82f3jc",
```

Comment:
we should use the recaptcha key you put in the config here instead of hard coding it. 

```suggestion
        siteKey: "process.env.RECAPTCHA_SITE_KEY",
```